### DISCUSSION TITLE
Mutli-layer support in VVenC?

### DISCUSSION THREAD

**Discussion by RedaKaafarani1** [fraunhoferhhi/vvenc#233] [General] [answered]
Hello,

Sorry if this was already asked by I couldn't find it anywhere else. Is there support for multi-layer encoding in VVenC now? It is supported in VTM but would be good to benefit from VVenC's speed increase, thanks in advance!

Best

**Comment by adamjw24** [comment, marked as answer]
VVenC for now does not support multi-layer. Some researchers from NHK did implement Multi-Layer support on top of an older version of VVenC, but so far I am not aware of them having published it. You can check out the technical report here: https://jvet-experts.org/doc_end_user/current_document.php?id=12013

**Reply by RedaKaafarani1** [reply]
Thank you @adamjw24, as you mentioned, I'm aware of the NHK implementation but it doesn't seem to be published yet, thanks for your input
